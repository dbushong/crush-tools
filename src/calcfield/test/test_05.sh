test_number=05
description="whitespace at line edges"

infile="$test_dir/edge_space.txt"
outfile="$test_dir/test_$test_number.actual"
expected="$test_dir/test_$test_number.expected"

$bin -d '\t' -e '[bar]+[baz]' -c tot $infile > "$outfile"

if [ $? -ne 0 ] ||
   [ "`diff -q $outfile $expected`" ]; then
  test_status $test_number 1 "$description" FAIL
else
  test_status $test_number 1 "$description" PASS
  rm "$outfile"
fi
