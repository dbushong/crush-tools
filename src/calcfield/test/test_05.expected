foo	bar	baz	tot
1	2	3	5
	4	5	9
